### PR TITLE
Moves the BGP session timeseries to TOML

### PR DESCRIPTION
**Pull request by bnaecker**

### CODE REVIEW

**Comment by bnaecker** [comment]
This is part of #5941, moving all the timeseries definitions into the new TOML format. I'll add a corresponding PR in Maghemite consuming this definition next.

File: oximeter/oximeter/schema/bgp-session.toml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+description = "Number of times our BGP peer has entered the open-sent state"
+units = "count"
+datum_type = "cumulative_u64"
+versions = [
+    { added_in = 1, fields = [ ] }
+]
+
+[[metrics]]
+name = "transition_to_session_setup"
+description = "Number of times our BGP peer has entered the open-sent state"
```
Comment (rcgoodfellow) [inline oximeter/oximeter/schema/bgp-session.toml:202]:
```suggestion
description = "Number of times our BGP peer has entered the session-setup state"
```

**Review by rcgoodfellow — COMMENTED** [review]
Just one small correction.

File: oximeter/oximeter/schema/bgp-session.toml (hunk 1 above)
Comment (bnaecker) [reply oximeter/oximeter/schema/bgp-session.toml]: Good catch, thanks! Fixed in 60c82e491

**Review by rcgoodfellow — APPROVED** [review]

**Pull request merged**